TabloidCLI/UserInterfaceManagers/ColorManager.cs

```diff
@@ -0,0 +1,142 @@
+using System.Text;
+using System.Linq;
+
+using System.Threading.Tasks;
+
+namespace TabloidCLI.UserInterfaceManagers
+{
+    public class ColorManager : IUserInterfaceManager
+    {
+        private readonly IUserInterfaceManager _parentUI;
+        public ColorManager(IUserInterfaceManager parentUI)
+        {
+            _parentUI = parentUI;
+        }
+
+        public IUserInterfaceManager Execute()
+        {
+            Console.WriteLine("Color Menu");
+            Console.WriteLine(" 1) Choose background color");
+            Console.WriteLine(" 2) Choose font color");
+            Console.WriteLine(" 3) Choose theme");
+            Console.WriteLine(" 0) Go Back");
+
+            Console.Write("> ");
+            string choice = Console.ReadLine();
+            switch (choice)
+            {
+                case "1":
+                    ChooseBackground();
+                    return this;
+                case "2":
+                    ChooseFont();
+                    return this;
+                case "3":
+                    ChooseTheme();
+                    return this;
+                case "0":
+                    return _parentUI;
+                default:
+                    Console.WriteLine("Invalid Selection");
+                    return this;
+            }
+        }
+
+        private void ChooseBackground()
+        {
+            ConsoleColor[] consoleColors = (ConsoleColor[])ConsoleColor.GetValues(typeof(ConsoleColor));
+
+            for (int i = 0; i < consoleColors.Length; i++)
+            {
+                ConsoleColor color = consoleColors[i];
+                Console.WriteLine($" {color}");
+            }
+            Console.Write("> ");
+
+            string input = Console.ReadLine();
+            try
+            {
+                Console.BackgroundColor = (ConsoleColor)Enum.Parse(typeof(ConsoleColor), input, true);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Invalid Selection");
+            }
+        }
+
+        private void ChooseFont()
+        {
+            ConsoleColor[] consoleColors = (ConsoleColor[])ConsoleColor.GetValues(typeof(ConsoleColor));
+
+            for (int i = 0; i < consoleColors.Length; i++)
+            {
+                ConsoleColor color = consoleColors[i];
+                Console.WriteLine($" {color}");
+            }
+            Console.Write("> ");
+
+            string input = Console.ReadLine();
+            try
+            {
+                Console.ForegroundColor = (ConsoleColor)Enum.Parse(typeof(ConsoleColor), input, true);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Invalid Selection");
+            }
+        }
+        private void ChooseTheme()
+        {
+            string[] themes = new string[]
+            {
+                "Easter Egg", "Donkey Kong", "Ocean View", "Bumble Bee", "TheHamptons", "Branty"
```

Comment:
Please change this to "The Hamptons", otherwise. This is incredible, you guys did an awesome job!! 🥚 🦍🌊  🐝

Reply:
Done! Can I get an approval? :)

Reply:
Perfection!